generators/app/templates/template/package.json

```diff
@@ -3,7 +3,8 @@
   "scripts": {
     "watch": "onchange '**/*.d.ts' -e 'out/**' -- npm run build",
     "build": "typings bundle<%- ambient %> -o out",
-    "test": "ts-node test/test.ts"
+    "test": "ts-node test/test.ts",
+    "postinstall": "typings install"
```

Comment:
oh, I didn't realize that the `typings` CLI tool actually ran `npm install` on these packages!